### PR TITLE
CUMULUS-1777: Delete old tokens from AccessTokens table

### PR DESCRIPTION
**Pull request by markdboyd**
**Summary:** Summary of changes

Addresses [CUMULUS-1777: Delete old tokens from AccessTokens table](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1777)

## Changes

* Enable TTL for AccessTokens table using expirationTime field
* Make expirationTime a required property of access tokens model
* Update all code setting expirationTime values to set values using seconds from the epoch (which is required for TTL deletion of records to work)
* Update access tokens model to set an expirationTime on create() if no value is provided

## PR Checklist

- [x] Update CHANGELOG
- [x] Unit tests
- [ ] Adhoc testing

### CODE REVIEW

**Comment by markdboyd** [comment]
Need to think about how to handle the case of deleting the access tokens table while a user is logged to the dashboard

File: packages/api/tests/models/test-access-tokens-model.js [hunk 1]
```diff
@@ -52,14 +54,22 @@ test('create() suceeds with only an access token value', async (t) => {
...
-  t.is(accessTokenRecord.expirationTime, undefined);
+});
+
+test('create() sets an expirationTime if none is provided', async (t) => {
+  const { accessToken } = fakeAccessTokenFactory();
+  const accessTokenRecord = await accessTokenModel.create({ accessToken });
+
+  t.is(accessTokenRecord.accessToken, accessToken);
+  t.true(isNumber(accessTokenRecord.expirationTime));
+  t.true(new Date(accessTokenRecord.expirationTime * 1000) > new Date());
```
Comment (markdboyd) [inline packages/api/tests/models/test-access-tokens-model.js:72]: Test this against `moment().unix()`

File: CHANGELOG.md [hunk 2]
```diff
@@ -29,6 +29,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 - **CUMULUS-1682**
   - Moved all `@cumulus/ingest/parse-pdr` code into the `parse-pdr` task as it had become tightly coupled with that task's handler and was not used anywhere else. Unit tests also restored.
+- **CUMULUS-1777**
```
Comment (markdboyd) [inline CHANGELOG.md:32]: Add note about deleting table

Comment (nemreid) [inline CHANGELOG.md:32]: Add breaking changes or migration steps section for recommended deletion of table.

File: packages/api/tests/app/test-launchpadAuth.js [hunk 3]
```diff
@@ -24,7 +22,14 @@ const validateTokenResponse = {
 };
 
 let accessTokenModel;
+
 test.before(async () => {
+  process.env.oauth_user_group = 'GSFC-Cumulus';
+  process.env.OAUTH_PROVIDER = 'launchpad';
+  process.env.AccessTokensTable = randomId('AccessTokens');
+  process.env.system_bucket = randomId('bucket');
+  process.env.stackName = randomId('stack');
```
Comment (nemreid) [inline packages/api/tests/app/test-launchpadAuth.js:31]: Delete these in the `after`.

**Pull request merged**